.github/workflows/coverage.yml

```
@@ -62,22 +62,26 @@ jobs:
           cd workspace
           echo "[" > ../_temp_coverage.json
 
-          COUNT=$(yq '.repositories | length' ../repos.yaml)
-
           OLD_COVERAGE_JSON=$(cat ../gh-pages/coverage.json || echo "[]")
 
-          for i in $(seq 0 $((COUNT - 1))); do
-            REPO=$(yq ".repositories[$i].name" ../repos.yaml | tr -d '"')
+          # Iterate through all repository configuration files
+          for config_file in ../repos/*.yaml; do
+            # Skip if no files found
+            [ -e "$config_file" ] || continue
+
+            REPO=$(yq '.name' "$config_file" | tr -d '"')
```

We _shouldn't_ need the extra pipe to `tr` here.
```suggestion
            REPO=$(yq -r '.name' "$config_file")
```